pkg/component/controller/workerconfig/reconciler.go

```diff
@@ -143,7 +143,7 @@ func (r *Reconciler) Init(context.Context) error {
 			return err
 		}
 		return (&applier.Stack{
-			Name:      "k0s-" + constant.WorkerConfigComponentName,
+			Name:      fmt.Sprintf("k0s-%s-%s", constant.WorkerConfigComponentName, constant.KubernetesMajorMinorVersion),
```

Comment:
This string can be a constant, same with the `system:bootstrappers:...` below. With `fmt.Sprintf` it's regenerated/allocated at runtime. This is in `Init` so maybe it doesn't matter much though.